config/src/main/java/org/springframework/security/config/web/server/ServerHttpSecurity.java

```diff
@@ -1275,6 +1318,46 @@ public AuthorizeExchangeSpec access(ReactiveAuthorizationManager<AuthorizationCo
 		}
 	}
 
+	/**
+	 * Configures HTTPS redirection rules
+	 *
+	 * @author Josh Cummings
+	 * @since 5.1
+	 * @see #redirectToHttps()
+	 */
+	public class HttpsRedirectSpec {
```

Comment:
@rwinch Would you recommend this at the top level since likely other filters will want to use it too? e.g. `http.portMapper(PortMapper)` vs `http.redirectToHttps().portMapper(PortMapper)`